python/ray/rllib/dqn/dqn.py

```diff
@@ -203,6 +219,7 @@ def train(self):
     logger.dump_tabular()
 
     res = TrainingResult(
-        self.num_iterations, mean_100ep_reward, mean_100ep_length)
+        self.experiment_id.hex, self.num_iterations, mean_100ep_reward,
+        mean_100ep_length)
     self.num_iterations += 1
-    return res
+    return res, info
```

Comment:
Hm should we return just one json object? Note that we can support arbitrary "info" types using MapType: http://docs.aws.amazon.com/athena/latest/ug/ddl/create-table.html